src/Template/Layout/default.ctp

```diff
@@ -6,13 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!-- Base CSS -->
-    <?= $this->Html->css('CrudView.bootstrap.min');?>
-    <?= $this->Html->css('CrudView.bootflat.min');?>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
```

Comment:
I'd rather not include a gigantic js/css file in our repo - especially minified.

If we wanted, we could depend upon asset compress, and then I would be only *okay* with including unminified assets. If you want to work on that, feel free to do so. I will not though.